### PR TITLE
adding openshift, kubernetes and data science resources 

### PR DESCRIPTION
**Pull request by fawazsiddiqi**
Adding Kubernetes and Red Hat OpenShift resources, I think it would be a good addition to this repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -336,8 +336,14 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 - [Kubernetes - Lets learn the basic tenets](https://atharvacodes.netlify.app/kubernetes-docker/Basic-tenets/)
 - [Understanding Kubernetes: Developer's Guide](https://blog.thewiz.net/understanding-kubernetes-developers-guide)
+- [Kubernetes Hands On & Training](https://www.katacoda.com/courses/kubernetes)
```
Comment (shahednasser) [inline README.md:339]: Add a new line below this

File: README.md [hunk 2]
```diff
@@ -336,8 +336,14 @@ To add a resource or contribute, please check out our contribution guide [here](
...
 ---
 
+## OpenShift
+
+- [OpenShift Interactive Learning Portal](https://learn.openshift.com)
+- [OpenShift Tutorials](https://developers.redhat.com)
```
Comment (shahednasser) [inline README.md:345]: Add a new line below this

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hello @fawazsiddiqi thank you for your contribution. I just need you tiny amendments detailed below

**Comment by fawazsiddiqi** [comment]
Resolved for both and added a new resources for data science

**Comment by fawazsiddiqi** [comment]
PS fixed the issue with the data science commit

**Comment by shahednasser** [comment]
Great thanks!

**Pull request merged**